tools/build_config/build.ps1

```diff
@@ -188,6 +192,57 @@ function Invoke-Package {
   }
 }
 
+function Invoke-Docs {
+    (Get-Content "./build/CPackConfig.cmake" |
+      Where-Object {$_ -match 'CPACK_PACKAGE_FILE_NAME'}) -match '.*"Horace-([^"]+)".*'
+  $build_id = $Matches[1]
+
+  (Get-Content ./documentation/user_docs/docs/conf.py) |
+    Foreach-Object {
+        $_ -replace '(version|release) = .*', "`$1 = '${build_id}'"
+    } |
+      Set-Content  ./documentation/user_docs/docs/conf.py
```

Comment:
As you're setting the `version=` and `release=` lines in `conf.py` can you remove the "plausible" placeholder values currently in that file (`3.5.0`) and replace them with something that is clearly a placeholder?